dspace-api/src/main/java/org/dspace/browse/BrowserScope.java

```diff
@@ -354,7 +360,7 @@ public int getResultsPerPage()
      */
     public void setResultsPerPage(int resultsPerPage)
     {
-        if (resultsPerPage > -1 || (browseIndex != null && browseIndex.isTagCloudEnabled()))
```

Comment:
I worked with an older source, did the fix in commit
https://github.com/quatroelementos/DSpace/commit/3f74f3cc24c1a988381e8c0d25405c6fef45ca1f
thanks

On Tue, Feb 24, 2015 at 5:33 AM, helix84 <notifications@github.com> wrote:

> In dspace-api/src/main/java/org/dspace/browse/BrowserScope.java
> <https://github.com/DSpace/DSpace/pull/872#discussion_r25235500>:
>
> > @@ -354,7 +360,7 @@ public int getResultsPerPage()
> >       */
> >      public void setResultsPerPage(int resultsPerPage)
> >      {
> > -        if (resultsPerPage > -1 || (browseIndex != null && browseIndex.isTagCloudEnabled()))
>
> This change is essentially a revert of the previous commit. Is it
> intentional or were you accidentally working with older source?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DSpace/DSpace/pull/872/files#r25235500>.
>



-- 
Rodrigo Prado de Jesus
http://about.me/royopa
royopa@gmail.com